ligo/test/LICENSE

```diff
@@ -0,0 +1,19 @@
+MIT License Copyright (c) 2020 TQ Tezos
```

Comment:
AFAIU this file is not necessary and can be removed.

`reuse lint` seems to report all things being ok without it as well.

Reply:
This is about complaints of cabal, we refer to a license file from there and it was not present.

Reply:
Ah I see :ok_hand: